content/en/docs/concepts/configuration/taint-and-toleration.md

```diff
@@ -277,15 +278,14 @@ admission controller](https://git.k8s.io/kubernetes/plugin/pkg/admission/default
   * `node.kubernetes.io/unreachable`
   * `node.kubernetes.io/not-ready`
 
-This ensures that DaemonSet pods are never evicted due to these problems,
-which matches the behavior when this feature is disabled.
+This ensures that DaemonSet pods are never evicted due to these problems.
 
 ## Taint Nodes by Condition
 
 The node lifecycle controller automatically creates taints corresponding to
 Node conditions.
 Similarly the scheduler does not check Node conditions; instead the scheduler checks taints. This assures that Node conditions don't affect what's scheduled onto the Node. The user can choose to ignore some of the Node's problems (represented as Node conditions) by adding appropriate Pod tolerations.
-Note that `TaintNodesByCondition` only taints nodes with `NoSchedule` effect. `NoExecute` effect is controlled by `TaintBasedEviction` which is a beta feature and enabled by default since version 1.13.
+Note that `TaintNodesByCondition` only taints nodes with `NoSchedule` effect. `NoExecute` effect is controlled by `TaintBasedEviction`.
```

Comment:
You should reword this to avoid mentioning the names of the two now-graduated feature gates.

Reply:
I removed those two lines completely and move mention of each effect to corresponding section.